### ISSUE TITLE
Infinite stepsize integrating a function

### ISSUE THREAD

**Issue by mforets** [PerezHz/TaylorIntegration.jl#66]
For the context see [this discussion](https://github.com/JuliaReach/Reachability.jl/issues/146#issuecomment-478704552). In short, integrating a function such as

```julia
@taylorize function bball!(t, x, dx)
    dx[1] = x[2]
    dx[2] = -one(x[1]) # this one is constant
    return dx
end
```
with a taylor series in time whose order is larger than two (the exact solution has degree two), then `TaylorIntegration.stepsize!` returns a stepsize which is `Inf`.

This is maybe correct? Since one gets the exact solution for all `t`? So i don't know the expected action point here, but then some error happens down the line, in `TaylorModels.validated_integ`.

**Comment by lbenet** [comment]
Thanks @mforets  for opening this issue!

Indeed, the returned computed local solution is actually exact, which means that if the order is 4 or larges, the returned time step will be `Inf`. What we could do is check for such a `Inf` time step, and only in that case, use the last non-zero coefficient of the series to compute a finite step size.

cc @PerezHz

**Comment by PerezHz** [comment]
Perhaps we use could use TaylorSeries' [`findlast`](https://github.com/JuliaDiff/TaylorSeries.jl/blob/5e796420fdc432588606b407c560cac0b68474c1/src/auxiliary.jl#L229)? For example, [here](https://github.com/PerezHz/TaylorIntegration.jl/blob/4fcdb71974de522284f567f77531f48506770200/src/explicitode.jl#L113), instead of doing `ord = x.order`, we could do `ord = findlast(x)`

**Comment by lbenet** [comment]
Good idea! Could you test it?

**Comment by lbenet** [comment]
Closing; solved by #73
